inference-engine/ie_bridges/python/sample/classification_sample_async/classification_sample_async.py

```diff
@@ -96,17 +96,23 @@ def build_argparser():
 def main():
     log.basicConfig(format="[ %(levelname)s ] %(message)s", level=log.INFO, stream=sys.stdout)
     args = build_argparser().parse_args()
-    model_xml = args.model
-    model_bin = os.path.splitext(model_xml)[0] + ".bin"
 
     # Plugin initialization for specified device and load extensions library if specified
     log.info("Creating Inference Engine")
     ie = IECore()
     if args.cpu_extension and 'CPU' in args.device:
         ie.add_extension(args.cpu_extension, "CPU")
-    # Read IR
-    log.info("Loading network files:\n\t{}\n\t{}".format(model_xml, model_bin))
-    net = ie.read_network(model=model_xml, weights=model_bin)
+
+    # Read a model in OpenVINO Intermediate Representation (.xml and .bin files) or ONNX (.onnx file) format
+    model = args.model
+    model_bin = None
+    model_name, model_ext = os.path.splitext(model)
+    log.info(f"Loading network files:\n\t{model}")
+    if model_ext == ".xml":
+        # Read.bin weights for IR format only
```

Comment:
`# Read.bin weights for IR format only`
Add space between `Read` and `.bin`

Reply:
fixed